### ISSUE TITLE
Move 'Fedora Objects' Panel to Manage Files Page

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#5325]
# Description
Currently the "Fedora Objects" link on the object view page returns the Fedora objects listing for the record. This can be moved from object view to an edit page—there is no use case for this to be on the object page. Where should this go on the edit pages? Manage Files, but maybe that won't work?

**Comment by joncameron** [comment]
Seems like we can just move the URL for the endpoint for Fedora into the metadata section of the IIIF manifest going forward. I will update the done looks like.

**Comment by joncameron** [comment]
We could also just remove this from the UI, and expose it only as an endpoint. I would prefer to just remove this from the object page unless a use case pops up.

**Comment by masaball** [comment]
This panel looks to have already been removed as part of the work on #5323. It was an element of the `_metadata_display` partial, which was deleted in that PR, and the element was not pulled out and moved to a different part of the code.

@joncameron @elynema I looked over a lot of our documentation but wasn't sure where seemed best to mention the fedora tree endpoint. Any thoughts on what page would be best to add that to?

**Comment by elynema** [comment]
@masaball Yeah, I'm not sure either. Presumably on the Samvera wiki pages under developer documentation. We probably should hold this until Jon C. can look at it.

**Comment by joncameron** [comment]
I added a note about the Fedora tree in the "Avalon Metadata" page. That seemed like the best place to put it, although it could get reorganized if that page gets spruced up. But it should probably live somewhere in the metadata pages and/or collection manager guides.
